.gitignore

```
@@ -39,4 +39,13 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
 config/database.yml
+config/settings.local.yml
+config/settings/*.local.yml
+config/environments/*.local.yml
+
+
+test/application_system_test_case.rb
+test/controllers/static_page_controller_test.rb/
+test/
```

```suggestion
test/*
```
it will ignore all file in test folder